chapters/offenses.adoc

```diff
@@ -12,9 +12,9 @@ All minor offenses are listed below.
 A kick is aimless when after the ball touched a robot, it subsequently crossed the midline and then its opponent's goal line outside the goal without touching another robot. A kick-off kick cannot be aimless, as the ball is located at the midline and does therefore not cross it.
 
 ==== Lack Of Progress
-There is a lack of progress if only one team is allowed to <<Ball Manipulation, manipulate the ball>> (<<Kick-Off, kick-off>>, <<Direct Free Kick, direct free kick>>, <<Indirect Free Kick, indirect free kick>>, <<Penalty Kick, penalty kick>>) and does not bring the ball <<Ball In And Out Of Play, into play>> in time. The time limit is 5 seconds for <<Indirect Free Kick, indirect free kicks>> and <<Direct Free Kick, direct free kicks>> in division A and 10 seconds in all other cases.
+There is a lack of progress if only one team is allowed to <<Ball Manipulation, manipulate the ball>> (<<Kick-Off, kick-off>>, <<Direct Free Kick, direct free kick>>, <<Indirect Free Kick, indirect free kick>>, <<Penalty Kick, penalty kick>>) and does not bring the ball <<Ball In And Out Of Play, into play>> in time. The time limit is 5 seconds in division A and 10 seconds in division B.
 
-There is also a lack of progress if the ball is inside a team's <<Defense Area, defense area>> for 10 seconds, since the keeper is the only robot that is allowed to <<Ball Manipulation, manipulate the ball>>.
+There is also a lack of progress if the ball is inside a team's <<Defense Area, defense area>> for 5 seconds, since the keeper is the only robot that is allowed to <<Ball Manipulation, manipulate the ball>>.
```

Comment:
As @tobiasheineken pointed out, for this situation, a force start is not reasonable.

The autoRefs separate this already: There is a `KeeperHeldBall` and a `KickTimeout` game event. I think we should separate this in the rules as well and make it two different minor offenses.

Reply:
Still, we've lost the Indirect Freekick for KeeperHeldBall which makes me sad.

Reply:
You can create a new discussion in the respective PR. If there is enough evidence, that an indirect free kick is still reasonable (like in this situation), what about merging all free kicks to indirect free kicks? Would this solve the issue, or is there actually something else that makes you uncomfortable with this rule change?